src/dmd/expression.d

```diff
@@ -4749,11 +4749,11 @@ extern (C++) class UnaExp : Expression
 
         if (e1.op == TOKtype)
         {
-            error("incompatible type for (%s(%s)): cannot use '%s' with types", Token.toChars(op), e1.toChars(), Token.toChars(op));
+            error("incompatible type for `%s(%s)`: cannot use '%s' with types", Token.toChars(op), e1.toChars(), Token.toChars(op));
```

Comment:
Should do the '%s' to `%s` while you're at it!

Reply:
Yes, there are several messages in this PR that use single quotes instead of backticks.  Please change them to backticks also.

Reply:
I find the brackets worse as they nest. There are many uses of '%s', I think they should be replaced separately.

Reply:
So be it.